lib/sqitch-passwords.pod

```diff
@@ -125,6 +129,49 @@ Vertica does not currently support a password file.
 
 Firebird does not currently support a password file.
 
+=item Exasol
+
+Exasol allows configuring connection profiles for the 'exaplus' client:
+
+  > exaplus -u sys -p exasol -c localhost:8563 -wp flipr_test
+  EXAplus 6.0.4 (c) EXASOL AG
+
+  Profile flipr_test is saved.
+  > exaplus -profile flipr_test -q -sql "select current_timestamp;"
+
+  CURRENT_TIMESTAMP
+  --------------------------
+  2017-11-02 13:35:48.360000
+
+These profiles are stored in F<~/.exasol/profiles.xml>, readable only to the user
+by default.
+
+For ODBC connections from Sqitch, we can use connection settings in F<~/.odbc.ini>:
+
+  [flipr_test]
+  DRIVER = Exasol
+  EXAHOST = localhost:8563
+  EXAUID = sys
+  EXAPWD = exasol
+
+When combining the above, Sqitch doesn't need to know any credentials; they are
+stored somewhat safely in F<~/.exasol/profiles.xml> and F<~/.odbc.ini>:
+
+  > sqitch status db:exasol:flipr_test
+  # On database db:exasol:flipr_test
+  # Project:  flipr
+  # Change:   7ebad104723cfa46ec1e5425ae1bbcc188327c8e
+  # Name:     userflips
+  # Deployed: 2017-11-02 15:02:39 +0100
+  # By:       Johan WÃ¤rlander <johan.warlander@schibsted.com>
```

Comment:
Do you want your full email in docs?

Reply:
Thanks for spotting that, too quick with  copy-paste..

While we're looking at that output though, any idea why I'm not seeing "Wärlander" above? It looks good in the database when I query `committer_name`, both with `exaplus` and `isql`.